setup.cfg

```diff
@@ -55,6 +55,7 @@ jwst =
 hst =
     hstcal
```

Comment:
May want to remove the `[hst]` extra as it is not possible to `pip install hstcal`.  It can only be installed via a conda package.

```suggestion
```

Reply:
I did on the other pull request